TACC-Cloud/geoapi
# Task/DES-1932 Streetview Organizations and Project link

## Overview: ##
Adds functionality for mapillary organizations and allows user to associate streetview objects with projects.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1932](https://jira.tacc.utexas.edu/browse/DES-1932)

## Summary of Changes: ##
Added changes to the project, streetview to associate with each other and added support for organizations.

## Testing Steps: ##
1. Test with frontend branch TACC-cloud/hazmapper#32
2. Check that publishing works with organizations 
3. Check that users can link their streetview object with projects.

## Notes: ##
Streetview branch #46  should be merged first.
